lesson06/index.js

```diff
@@ -0,0 +1,40 @@
+const imageArray = [
+    {
+        to: "bookmark.html", 
+        img: "1.png", 
+        alt:"画像1", 
+        text: "ブックマーク"
+    }, 
+    {
+        to: "message.html", 
+        img: "2.png", 
+        alt:"画像2", 
+        text: "メッセージ"
+    }
+];
+
+const ul = document.getElementById('js-list');
+const fragment = document.createDocumentFragment();
+
+new Promise((resolve) => {
+    setTimeout(() => {
+        resolve(imageArray);
+    }, 3000);
+
+}).then((val) => {
+    for (let i = 0; i < val.length; i++) {
+        const li = document.createElement('li');
+        const a = document.createElement('a');
+        const img = document.createElement('img');
+    
+        a.href = `/${val[i].to}`;
+        a.textContent = `${val[i].text}`;
+        img.src = `${val[i].img}`;
+        img.alt = `${val[i].alt}`;
```

Comment:
question

I felt that the three codes I selected did not need the template literal.
If you have a reason why they are needed, I'd love to hear it!

I wondered if I could do this↓
```  
    a.textContent = val[i].text;
    img.src = val[i].img;
    img.alt = val[i].alt;
```

Reply:
Thank you for review!
As you said, the template literal was unnecessary, so I removed it.
[a382856](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/6/commits/a382856b9f9bba1a3380b59dff80b1fc84d277bc)

Reply:
I'm sorry after I received your approval.
Added a href template literal.
[7a2a0f7](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/6/commits/7a2a0f70a24f3dd8d2db31094b93e144f3c6f006)
[CodeSandBox](https://codesandbox.io/s/lesson06-f6p77)